draft-ietf-rtcweb-jsep.xml

```diff
@@ -790,16 +790,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           of the MediaStreamTrack in order to satisfy the limits.
           Downscaling MUST NOT change the track aspect ratio.</t>
 
-          <t>If the original resolution is less than the size limits in
-          the attribute, upscaling is needed, but this may not be
-          appropriate in all cases. To address this concern, the
-          application can set an upscaling policy for each sent track.
-          For this case, if upscaling is permitted by policy, the
-          sender SHOULD apply upscaling in order to provide the desired
-          resolution. Otherwise, the sender MUST NOT apply upscaling.
-          The sender SHOULD NOT upscale in other cases, even if the
-          policy permits it. Upscaling MUST NOT change the track aspect
-          ratio.</t>
+          <t>If the original resolution is less than the size limits
```

Comment:
This was in the original, but maybe "less than that indicated in the size limits".

It's also weird to compare paired quantities like this as if they were scalars.
Is 1024x768 smaller than 1000x1000?